I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 16, in script
    k_nvl "you called?"
  File "renpy/common/00nvl_mode.rpy", line 381, in do_display
    **display_args)
IOError: Couldn't find file 'audio/ReceiveText.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 16, in script
    k_nvl "you called?"
  File "renpy/ast.py", line 715, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "renpy/exports.py", line 1393, in say
    who(what, *args, **kwargs)
  File "renpy/character.py", line 1205, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "renpy/common/00nvl_mode.rpy", line 381, in do_display
    **display_args)
  File "renpy/character.py", line 621, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3101, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3794, in interact_core
    renpy.audio.audio.periodic()
  File "renpy/audio/audio.py", line 1101, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "renpy/audio/audio.py", line 1079, in periodic_thread_main
    periodic_pass()
  File "renpy/audio/audio.py", line 1021, in periodic_pass
    c.periodic()
  File "renpy/audio/audio.py", line 487, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "renpy/loader.py", line 798, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'audio/ReceiveText.ogg'.

Windows-10-10.0.19041
Ren'Py 7.4.4.1439
talk to me 1.0
Sat Jul  2 00:13:58 2022
